include/git2/common.h

```diff
@@ -372,6 +374,18 @@ typedef enum {
  *		> fail.  (Using the FORCE flag to checkout will still overwrite
  *		> these changes.)
  *
+ *	 opts(GIT_OPT_GET_INDEXER_MAX_OBJECTS, size_t *out)
```

Comment:
I think I'd rather name this `GIT_OPT_{SET,GET}_PACK_MAX_OBJECTS`. Mentioning the indexer here is simply leaking implementation details which are not of much interest to the general user.